doc/api/console.md

```diff
@@ -250,6 +290,15 @@ If formatting elements (e.g. `%d`) are not found in the first string then
 [`util.inspect()`][] is called on each argument and the resulting string
 values are concatenated. See [`util.format()`][] for more information.
 
+### console.table([data][, ...args])
+<!-- YAML
+added: REPLACEME
+-->
+* `data` {any}
+* `...args` {any}
+
+Alias for `console.log([data[, ...args]]).
```

Comment:
oh, I see what you're referring to. No, that was just a misplaced `]` on my part :-)